test/scripts/boot-image

```diff
@@ -86,6 +86,7 @@ def boot_ami(distro, arch, image_type, image_path):
                    "--username", "osbuild",
                    "--ssh-privkey", privkey,
                    "--ssh-pubkey", pubkey,
+                   "--boot-mode" , "uefi",
```

Comment:
This is fine but we might want to bios boot some images where supported (our AMIs actually support both).